src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -29,6 +29,7 @@ const {
 } = sharedListMethods;
 
 /**
+ * @slot default - A slot for adding pick-list-item elements or pick-list-groups elements. Items are displayed as a vertical list.
```

Comment:
Looks like Ionic doesn't provide a name for the default slot: https://github.com/ionic-team/ionic/blob/34dfc3ce98d1e30af7e2f82276a63bc077726773/core/src/components/button/button.tsx#L12 I think we should follow suit to avoid a "default" named slot in the doc.

Reply:
Hmmm. Maybe there's a better way to convey this info, or a better name? I feel it's valuable information to provide to anyone who's using one of these components for the first time, to both understand the required props and the expected children. It's like needing to know that if you're using a <select> tag, you need to fill it with <option> elements that have `value` attributes.

Reply:
I think the "usage" folder will help out here once @kat10140 can get to those doc examples

Reply:
I'm not denying that it's valuable. It's just that the current tooling will generate a string field for the slot name when provided and could be misleading.

Reply:
I agree, we don't want to put a name here. Otherwise users may do this: `<div slot="default"></div>`

Reply:
Ahhhh so you're saying just do it like this: ` * @slot - A slot for...`?

Reply:
Yes.